hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -237,6 +237,9 @@ public Operation convert(String value) throws ParameterException {
     @Parameter(names = {"--enable-hive-sync"}, description = "Enable syncing to hive")
     public Boolean enableHiveSync = false;
 
+    @Parameter(names = {"--hoodie-sync-client-tool-class"}, description = "Meta sync client tool, using comma to separate multi tools")
```

Comment:
Should we change `--enable-hive-sync` to `--enable-sync` above?

Reply:
This place is worth discussing. For compatible, we just add a --hoodie-sync-client-tool-class.
But i think use change --enable-hive-sync to   --enable-sync reasonable , or just add a new parameter --enable-sync and compatible the --enable-hive-sync parameter for old users. cc @vinothchandar @leesf

Reply:
yes, `--enable-sync` is reasonable since hudi will not only supports sync to hive, but also other meta @garyli1019 . but as @lw309637554 pointed out, the compatible is a problem here, so the current solution is compromised solution. I am ok to rename to `--enable-sync` and document the break change here.

Reply:
ok , i  can add a new --enable-sync  as the default choice , and also support --enable-hive-sync  for a duplicate parameter.  what do you think about ? @garyli1019 @leesf

Reply:
sounds good to me

Reply:
done

Reply:
I think we can name this little shorter. `--sync-tool-class-list` ?